value.go

```diff
@@ -1275,6 +1275,9 @@ func bytesToKeyValue(pType int, buf []byte, offset int, length int) (Value, Erro
 		}
 		return ListValue(v), nil
 
+	case ParticleType.NULL:
+		return NewNullValue(), nil
```

Comment:
I believe this should return `nil, nil`.
`NullValue` implies the value was returned from the server, which is not possible.

Reply:
Updated.